tests/TestCase/View/Helper/ToolbarHelperTest.php

```diff
@@ -78,10 +79,30 @@ public function testDumpCoerceHtml()
             Debugger::configInstance('exportFormatter'),
             'Should restore setting'
         );
+
         // Restore back to original value.
         Debugger::configInstance('exportFormatter', $restore);
     }
 
+    public function testDumpSorted()
+    {
+        $getValues = function ($el) {
+            return (string)$el;
+        };
```

Comment:
Use `strval()` instead

Reply:
ok.

Reply:
Why that?

Reply:
Whole closure can be replaced with PHP function, not code in closure.